ui/src/App.less

```diff
@@ -239,3 +238,10 @@ body {
     }
   }
 }
+
+.ant-message {
```

Comment:
I assume this is the default class coming directly from ant. Can we have a custom class for the overrides? In case we add another component that uses this class, these changes will take effect without noticing and they are very specific to this usecase

Reply:
Yes, the problem is that without some of these settings (especially the z-index) the messages don't show (as they are covered by the header and the footer), and anyway since we have the header visible in all pages we can't really put this at the top since it would look a bit messy over the header, so I think if we use messages for this or any other other use case (and to have some consistency) it would be better to have them at the bottom